pm-commotion-mac2ip

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+#
+# Script written for PittMesh by Jason Khanlar
+#
+
+# Check for --listall, otherwise proceed
+
+if [ "$*" = "--listall" ]; then
+  mac1=DC
+  mac2=9F
+  mac3=DB
+  mac4=00
+  mac5=00
+  mac6=00
+
+  ip1=100
+  ip2=64
+  ip3=0
+  ip4=0
+
+  for ip2 in `seq 0 255`;do
+    #((ip2s63=ip2-63))
+    #((ip2=ip2-63))
+
+    mac4=`echo "obase=16; $ip2" | bc`
+
+    ((ip2=(ip2+64)%64 + 64))
+
+#    # Adjust ip 2nd octet to cycle through 64-127 four times
+#    if [ $ip2 -ge 256 ]; then
+#      ((ip2-=192))
+#    elif [ $ip2 -ge 192 ]; then
+#      ((ip2-=128))
+#    elif [ $ip2 -ge 128 ]; then
+#      ((ip2-=64))
+#    fi
+#    ip2=$((ip2 % 64 + 63))
+
+    # Pad with 0
+    if [ ${#mac4} -eq 1 ]; then mac4=0$mac4; fi
+
+#    for ip3 in `seq 255`;do
```

Comment:
While it might be useful to have the history of things here, please do remove unnecessary code that's been commented out. You can just do it in another commit on top instead of amending this commit.